src/config.rs

```diff
@@ -0,0 +1,275 @@
+use serde::{Deserialize, Serialize};
+use std::error;
+use std::io::Write;
+use uuid::Uuid;
+
+use tiny_http::{Response, Server};
+
+use std::collections::HashMap;
+use url::Url;
+
+use serde_json::Value;
+use tungstenite::{connect, Message};
+
+use ha_api::types::{GetAccessTokenResponse, RegisterDeviceResponse};
+use ha_api::HomeAssistantAPI;
+
+// HA creates tokens for 10 years so we do the same
+const LONG_LIVED_TOKEN_VALID_FOR: u32 = 365;
+
+const LONG_LIVED_TOKEN_WS_COMMAND_ID: u32 = 11;
+
+const LOCAL_SERVER_HOST: &str = "0.0.0.0:8000";
```

Comment:
I think we actually only want this to bind to an internal interface.

```suggestion
const LOCAL_SERVER_HOST: &str = "127.0.0.1:8000";
```

Reply:
Can do.